### PR TITLE
Implement quad pixels characters for drawing bitmaps

### PR DESCRIPTION
**Pull request by tomlm**
The old algo mapped every pixel to a character. It looked like this:
image

The new algo maps every 4 pixels to a single character using block characters 
 ▘▝▖▗ ▚ ▞ ▌ ▐ ▄ ▀ ▛ ▜ ▙ ▟ █
This effectively gives us 4x the resolution of the old algo as every 4x4 pixel is mapped to a single character + backgroundColor and foregroundColor. 

Images now look like this:
image

## Summary by CodeRabbit

- **New Features**
	- Enhanced bitmap rendering with support for quad pixel mapping, improving visual representation.
	- Improved handling of tab and newline characters in text rendering.
	- Introduced a new constructor for `PixelBackground` to simplify color initialization.
	- Updated image source in the gallery to a higher-quality PNG format.

- **Bug Fixes**
	- Refined pixel blending logic for better color accuracy in rendered output.

### CODE REVIEW

**Comment by tznind** [comment]
This is very cool! I recently added [Sixel support to Terminal.Gui](https://github.com/gui-cs/Terminal.Gui/pull/3734) - this could make a great fallback approach for terminals that do not support it.

**Comment by tomlm** [comment]
Sexsel looks cool! I was looking for something like that but didn't find it!

File: src/Consolonia.Core/Drawing/DrawingContextImpl.cs [hunk 1]
```diff
@@ -59,22 +60,42 @@
...
+            canvas.DrawBitmap(bmp.Bitmap, new SKRect(0, 0, bitmap.Width, bitmap.Height),
                 new SKPaint { FilterQuality = SKFilterQuality.Medium });
 
-            // Rect clip = CurrentClip.Intersect(destRect);
-            int width = bitmap.Info.Width;
-            int height = bitmap.Info.Height;
-            for (int x = 0; x < width; x++)
-            for (int y = 0; y < height; y++)
+            for (int y = 0; y < bitmap.Info.Height; y += 2)
+            for (int x = 0; x < bitmap.Info.Width; x += 2)
```
[the remark this answers is not in the document]
Comment (tomlm) [reply src/Consolonia.Core/Drawing/DrawingContextImpl.cs:71]: since we scale by *2 we are gaurenteed to have even pixels

**Comment by tomlm** [comment]
I also implement transparency for bitmaps
image

**Review by jinek — APPROVED** [review]

**Pull request merged**